kafka/protocol/admin.py

```diff
@@ -619,7 +619,7 @@ class CreatePartitionsResponse_v0(Response):
     API_VERSION = 0
     SCHEMA = Schema(
         ('throttle_time_ms', Int32),
-        ('topic_errors', Array(
```

Comment:
Java docs calls this field `topic_errors` for the `CreateTopics` and `CreatePartitions` calls. Only `DeleteTopics` returns `topic_error_codes` so that's why I'd handled the difference in the `_send_request_to_controller()`

Reply:
I cleaned up the `CreateTopicsResponse` classes to match Java and use `topic_errors`.